suite-common/formatters/src/formatters/convertCryptoToFiatAmount.ts

```diff
@@ -0,0 +1,25 @@
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { FiatRates } from '@suite-common/wallet-types';
+import { FiatCurrencyCode } from '@suite-common/suite-config';
+
+type CryptoToFiatAmountValue = string | null;
+
+export const convertCryptoToFiatAmount = ({
```

Comment:
tests for this

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/9e1594b2c248d5b2600ce14631665d76d4a64eed